Testscripts/Linux/nested_kvm_basic_test.sh

```diff
@@ -43,26 +52,28 @@ done
 ImageName="nested.qcow2"
 
 if [ -z "$NestedImageUrl" ]; then
-	echo "Please mention -NestedImageUrl next"
-	exit 1
+	LogErr "Please mention -NestedImageUrl next"
```

Comment:
"Missing the required parameter -NestedImageUrl. Test Aborted"
Please apply to below 4 condition blocks

Reply:
Updated. Thanks!